modular_skyrat/master_files/code/modules/client/preferences/erp_preferences.dm

```diff
@@ -53,11 +53,19 @@
 		return FALSE
 	. = ..()
 
+/datum/preference/toggle/erp/apply_to_client_updated(client/client, value)
+	. = ..()
+	if(!value && ishuman(client.mob))
+		var/mob/living/carbon/human/target = client.mob
```

Comment:
This would allow having a nullcheck and only having to use the client variable once.
```suggestion
	var/mob/living/carbon/human/target = client?.mob
	if(!value && istype(target))
```